privatek8s.tf

```
@@ -32,7 +30,7 @@ resource "azurerm_kubernetes_cluster" "privatek8s" {
   role_based_access_control_enabled = true # default value, added to please tfsec
   api_server_authorized_ip_ranges = setunion(
     values(local.admin_allowed_ips),
-    data.azurerm_subnet.vpn.address_prefixes,
+    data.azurerm_subnet.data_tier.address_prefixes,
```

```suggestion
    data.azurerm_subnet.private_vnet_data_tier.address_prefixes,
```
See [jenkins-infra/azure-net@`8890567` (#27)](https://github.com/jenkins-infra/azure-net/pull/27/commits/889056785314f3bf63307ef77e7869a4c8767f52)